gradle.properties

```diff
@@ -1 +1,2 @@
 version=3.0.0.BUILD-SNAPSHOT
+org.gradle.jvmargs=-Xmx4g
```

Comment:
I'll remove this when merging. JVM opts are set in `gradlew` and `gradlew.bat` scripts.

Reply:
I removed that. I forgot to remove that before committing